src/api/getRequests.ts

```diff
@@ -4,6 +4,10 @@ const getParticipants = async () => {
     return await axios.get("http://localhost:3001/participants");
 };
 
+const getParticipant = async (_name: string) => {
+    return await axios.get(`http://localhost:3001/participants?name=${_name}`);
+};
+
```

Comment:
you didn't type the response
`const getParticipant: (name: string) => Promise<_someType>`